tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -1292,6 +1293,38 @@ remove_dead_instrs(void)
   }
 }
 
+/**
+   \brief process debug info of variables with parameter attribute.
+ */
+void
+process_params(void)
+{
+  unsigned smax = stb.stg_avail;
+  for (SPTR sptr = get_symbol_start(); sptr < smax; ++sptr) {
+    if (STYPEG(sptr) == ST_PARAM && should_preserve_param(sptr)) {
+      DTYPE dtype = DTYPEG(sptr);
+      if (DTY(dtype) == TY_ARRAY || DTY(dtype) == TY_STRUCT) {
+        /* array and derived types have 'var$ac' constant variable
+         * lets use that, by renaming that to 'var'.
+         */
+	SPTR new_sptr = (SPTR) CONVAL1G(sptr);
+	NMPTRP(new_sptr, NMPTRG(sptr));
+      } else {
+	LL_DebugInfo *di = cpu_llvm_module->debug_info;
```

Comment:
Indentation looks strange here.

Reply:
Thanks for pointing this out. I have now using clang-format.